### PR TITLE
add tsconfig

### PR DESCRIPTION
**Pull request by itajaja**

### CODE REVIEW

File: packages/tsconfig-4catalyzer/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "tsconfig-4catalyzer",
+  "version": "0.1.0",
+  "main": "tsconfig.base.json",
```
Comment (itajaja) [inline packages/tsconfig-4catalyzer/package.json]: not sure what to put here 🤔

File: packages/tsconfig-4catalyzer/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "tsconfig-4catalyzer",
```
Comment (itajaja) [inline packages/tsconfig-4catalyzer/package.json]: is the name good?

**Comment by itajaja** [comment]
@jquense 	I have no idea how to deploy this 😆 can you help me? I'd like to use this in @4c/cli

File: packages/tsconfig-4catalyzer/package.json (hunk 1 above)
Comment (itajaja) [reply packages/tsconfig-4catalyzer/package.json]: maybe `@4c/tsconfig`?

File: packages/tsconfig-4catalyzer/package.json (hunk 2 above)
Comment (jquense) [reply packages/tsconfig-4catalyzer/package.json]: Maybe `@4c/tsconfig`, can we add a workspaces base version in this package as well, like just an additional file to cherry pick from take from frontend

Comment (itajaja) [reply packages/tsconfig-4catalyzer/package.json]: should I just add `"plugins": [{ "name": "typescript-workspace-plugin" }]`?

File: packages/tsconfig-4catalyzer/tsconfig.web.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
[the remark this answers is not in the document]
Comment (itajaja) [reply packages/tsconfig-4catalyzer/tsconfig.web.json]: smarty

File: packages/tsconfig/package.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@4c/tsconfig",
+  "version": "0.1.0",
+  "main": "base.json",
```
Comment (itajaja) [inline packages/tsconfig/package.json]: @jquense please double check these values to make sure they are actually correct 🙏

Comment (jquense) [reply packages/tsconfig/package.json]: yup looks fine

**Pull request merged**